### DISCUSSION TITLE
Side-by-side citations not formatting correctly?

### DISCUSSION THREAD

**Discussion by josephpusey** [Zettlr/Zettlr#3394] [Q&A]
I'm greatly enjoying using Zettlr and Zotero for the first time for a literature review, having been previously dependent on the comparatively clunky Mendely plugin for Word.

However, I'm having an issue with Zettlr in which citations placed next to each other are formatting incorrectly- please see screenshots.

<img width="613" alt="image" src="https://user-images.githubusercontent.com/88504612/163390292-21953b4e-fdf6-4d4f-99d7-351af6e18dc3.png">

compiles to pdf as 

<img width="805" alt="image" src="https://user-images.githubusercontent.com/88504612/163390371-448ceb1b-170f-4335-bbdc-f401a8931532.png">

What I want is something more like the Mendely plugin gives- blah blah blah [1,2] or blah blah blah [1-3]. Is there some setting I'm missing/formatting trick that would enable this?

Many thanks.

**Comment by nathanlesage** [comment]
What you are looking for is a way to cite multiple sources at once. Instead of writing `@citekey1 @citekey2 @citekey3` write `[@citekey1; @citekey2; @citekey3]` (the semicolons are important)

Writing just the citekey is meant for in-text citations that will then render to `As author (year) said ...` (given that you have an in-text citation style, of course)
